doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1260,7 +1262,8 @@ Performance Improvements
 Documentation Changes
 ~~~~~~~~~~~~~~~~~~~~~
 
--
+- Added sphinx spelling extension, updated documentation on how to use the spell check (:issue:`21079`)
```

Comment:
why was this changed? I think these are just old / rebase issues.

Reply:
This was a bad merge that I'm going to revert

Reply:
Looks like the revert didn't work as intended so I'm going to need to fix that as well